src/installPkg.js

```diff
@@ -4,13 +4,13 @@ const exec = child_process.exec;
 
 const installPkg = (pkgName, { cwd = process.cwd() }) => {
   return new Promise((resolve, reject) => {
-    const spinner = ora(`Installing "${pkgName}"...`).start();
+    console.log(`Installing "${pkgName}"...`);
```

Comment:
@chihsuan I don't think this is a good UI & UX
please refer to how `now-cli` does.

```
➜  hanata git:(master) now
> Deploying ~/Documents/canner/hanata under chilijung
> Synced 3 files (29.4KB) [2s]
> Using Node.js 8.11.3 (default)
> https://hanata-customize-isjsqqnjum.now.sh [in clipboard] (sfo1) [2s]
> Building…
```
`> ` should be color gray.

Reply:
Updated.